submissions/sprint_c/test_helpers.py

```diff
@@ -0,0 +1,198 @@
+import unittest
+import os
+import shutil
+from flask import json
+from helpers import FileManager, Vocabulary, EpithetGenerator
+
+
+class HelperTests(unittest.TestCase):
+
+    path = './temp_dir/data.json'
+    sad_path = './fake_dir/data.csv'
+
+    def setUp(self):
+        data = {
```

Comment:
You've defined data in setUp but didn't assign it as an instance variable of the test. You can avoid repeating this dictionary in other tests by assigning it to self.data, or you can create a fixture passed to the test methods: https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments